test/react/functional-components/simple-wth-object-assign2.js

```diff
@@ -0,0 +1,26 @@
+var React = require('react');
+// the JSX transform converts to React, so we need to add it back in
+this['React'] = React;
+
+function A(props) {
+  var copyOfProps = Object.assign({}, props.bag);
+  return <div>Hello {copyOfProps.x}</div>;
+}
+
+function App(props) {
+	var copyOfProps = Object.assign({}, props);
```

Comment:
Nit: tabs/spaces

Reply:
I have no idea why this project does this for VS Code, Sublime and Atom, I'm so confused why it only happens for Prepack.